asyncapi-client.gemspec

```diff
@@ -16,21 +16,21 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib,spec/factories}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
-  s.add_dependency "rails", "> 6"
+  s.add_dependency "rails", '~> 7.2'
```

Comment:
unpin this and add appraisal gem with appraisals for rails 7.2 and 8.0... 7.1 is EOL in 2 weeks so you probably don't need to bother with that one